dock/run.sh

```diff
@@ -10,7 +10,7 @@ folders="`pwd` /Users /home"
 #folder_maps="build node_modules"
 
 #absolute prefix, where following folders are placed.
-DOCKPREFIX=~/build/$DOCKANE-folders/
```

Comment:
good catch... but the DOCKPREFIX is actually not in use, and should be removed

Reply:
ok.
let me know what's wrong with the docker build. we got it to work on macs and linux machines here.